config/jenkins_infra.ci.jenkins.io.yaml

```diff
@@ -669,6 +669,19 @@ controller:
               whiteB: "#FFFFFF"
               yellow: "#CDCD00"
               yellowB: "#FFFF00"
+      custom-header: |
+        unclassified:
+          customHeaderConfiguration:
+            enabled: true
+            header: "logo"
+            headerColor:
+              backgroundColor: "linear-gradient(90deg, rgba(239,192,92,1) 0%, rgba(171,28,28,1) 100%);"
+              color: "white"
+              hoverColor: "grey"
+            logo:
+              svg:
```

Comment:
svg doesn't seem to render well in dark mode, png looks a lot better
<img width="236" alt="image" src="https://github.com/jenkins-infra/kubernetes-management/assets/21194782/bdc1d281-2b60-4840-a81a-0c3b418bcb20">